### DISCUSSION TITLE
423The account is blocked

### DISCUSSION THREAD

**Discussion by rufana-mirzaliyeva** [emailjs-com/help#12] [Q&A]
It has error like 423The account is blocked when I try to create email service. I tried to disconnect and connect again but it doesn't work. I need help to find issue

**Comment by xr0master** [comment]
An account can be blocked if suspicious behavior is observed. For example, too intensive requests that do not meet the account quota. Usually, this implies a dead loop in the code - the code has a bug and is not written correctly. Either the client is trying to DDoS us.

If the account has just been created, and you know for sure that you didn’t have time to do anything suspicious, then this may be some kind of issue on our side. I suggest refreshing the API keys in this case (in the Account page) and refreshing the browser page after that.

You can also contact support so that they check what the issue is, but this will take time.
